NineChronicles.Headless/AccountStateGetterExtension.cs

```diff
@@ -0,0 +1,50 @@
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Nekoyume.Action;
+using Nekoyume.Model.State;
+using static Lib9c.SerializeKeys;
+
+namespace NineChronicles.Headless
+{
+    public static class AccountStateGetterExtension
+    {
+        public static AvatarState GetAvatarState(this AccountStateGetter accountStateGetter, Address avatarAddress)
+        {
+            if (accountStateGetter(avatarAddress) is Dictionary dictionary)
+            {
+                string[] keys =
+                {
+                    LegacyInventoryKey,
+                    LegacyWorldInformationKey,
+                    LegacyQuestListKey,
+                };
+
+                bool backward = false;
```

Comment:
Is this a flag representing the avatar before the states were diveded?

Reply:
yes

Reply:
`backward` isn't direct naming for that IMO, Is there any other idiom for the (previously) unified avatar state? (e.g. v1?)